recipes/boost-ext-ut/all/test_package/CMakeLists.txt

```diff
@@ -1,14 +1,11 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
quite high for c3i, I think some runners may have CMake 3.15 only.

for C++20, CMake 3.12 is sufficient

Reply:
Yes, I've run in to that. I've bumped to CMake 3.15 for now to be safe.